src/main/java/cm/ptks/craftflowers/storage/LanguageFile.java

```diff
@@ -0,0 +1,472 @@
+        FLOWER_POTATO_AGE_4 = language.getString("flower.POTATO_AGE.POTATO_4");
+        FLOWER_POTATO_AGE_5 = language.getString("flower.POTATO_AGE.POTATO_5");
+        FLOWER_POTATO_AGE_6 = language.getString("flower.POTATO_AGE.POTATO_6");
+        FLOWER_POTATO_AGE_7 = language.getString("flower.POTATO_AGE.POTATO_7");
+
+        FLOWER_CARROT = language.getString("flower.CARROT");
+        FLOWER_CARROT_AGE_0 = language.getString("flower.CARROT_AGE.CARROT_0");
+        FLOWER_CARROT_AGE_1 = language.getString("flower.CARROT_AGE.CARROT_1");
+        FLOWER_CARROT_AGE_2 = language.getString("flower.CARROT_AGE.CARROT_2");
+        FLOWER_CARROT_AGE_3 = language.getString("flower.CARROT_AGE.CARROT_3");
+        FLOWER_CARROT_AGE_4 = language.getString("flower.CARROT_AGE.CARROT_4");
+        FLOWER_CARROT_AGE_5 = language.getString("flower.CARROT_AGE.CARROT_5");
+        FLOWER_CARROT_AGE_6 = language.getString("flower.CARROT_AGE.CARROT_6");
+        FLOWER_CARROT_AGE_7 = language.getString("flower.CARROT_AGE.CARROT_7");
+
+        FLOWER_BARRIER = language.getString("flower.BARRIER");
+
+        // item
+        ITEM_SECOND_PREFIX = language.getString("item.second_prefix");
+
+        // gui
+        GUI_TITLE = language.getString("gui.title");
+        GUI_CREATE_FLOWER = language.getString("gui.create_flower");
+        GUI_CANT_CREATE_EMPTY_FLOWER = language.getString("gui.cant_create_empty_flower");
+        GUI_REMOVE_FROM_TOP = language.getString("gui.remove_from_top");
+        GUI_CLEAR_ALL = language.getString("gui.clear_all");
+        GUI_FLOWER_GROUP_TITLE_COLOR = language.getString("gui.flower_group_title_color");
+        GUI_GLASS_NAME_COLOR = language.getString("gui.glass_name_color");
+        GUI_PREVIOUS = language.getString("gui.previous");
+        GUI_NEXT = language.getString("gui.next");
+        GUI_SAVED_FLOWERPOTS = language.getString("gui.saved_flowerpots");
+        GUI_SAVED_FLOWERPOTS_NAME_COLOR = language.getString("gui.saved_flowerpots_name_color");
+
+        // action
+        ACTION_NO_PERMISSION_PLACE = language.getString("action.no_permission_place");
+        ACTION_MISSING_FOLLOWING_ITEMS = language.getString("action.missing_following_items");
+        ACTION_MISSING_FOLLOWING_ITEMS_LIST = language.getString("action.missing_following_items_list");
+        ACTION_NO_PERMISSION_EDIT = language.getString("action.no_permission_edit");
+        ACTION_OUTDATE_VERSION = language.getString("action.outdate_version");
+        ACTION_CURRENT_VERSION = language.getString("action.current_version");
+        ACTION_NEWEST_VERSION = language.getString("action.newest_version");
+        ACTION_DOWNLOAD_NEW_VERSION = language.getString("action.download_new_version");
+        ACTION_FAILED_CHECK_VERSION = language.getString("action.failed_check_version");
+
+    }
+
+    public static String getCommandVersion(String version, String color) {
+        return COMMAND_VERSION.replaceAll("%1", color + version);
```

Comment:
This counts for all the strings that are getting variables replaced in this file:
Can't you just do "\<version\>" for example? Since you are already using replace all, this would be easy to implement, and it would be easier for translators